src/OrbitGl/Batcher.cpp

```diff
@@ -275,7 +275,7 @@ void Batcher::AddTriangle(const Triangle& triangle, const std::array<Color, 3>&
   user_data_.push_back(std::move(user_data));
 }
 
-Vec3 Batcher::TransformVertex(const Vec3 input) const {
+Vec3 Batcher::TransformVertex(const Vec3& input) const {
```

Comment:
I wonder if this is really a performance improvement, passing by value should use xmm registers, have you observed that this was indeed an optimization?

Reply:
Hm that's a good point, I didn't think of this. I didn't specifically measure this, I just noticed it and figured it would make sense to change it, I haven't explicitly measured the impact. It shouldn't hurt though, should it?

Reply:
Should be fine